content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -0,0 +1,138 @@
+variable. For example, to set it globally for all of a user's projects:
+
+```cli
+$ dvc config --global studio.token ***
+```
+
+While the experiment runs, you will see live updates like this in DVC Studio:
+
+![Notebook](/img/sagemaker-notebook.gif)
+
+## Pipelines
+
+You can run SageMaker jobs in DVC [pipelines] or convert existing SageMaker
+pipelines into DVC pipelines. This combines the benefits of SageMaker jobs, like
+running each stage on its own EC2 instance and enabling other data
+[input modes](https://docs.aws.amazon.com/sagemaker/latest/dg/model-access-training-data.html),
+with the benefits of DVC pipelines, like skipping unchanged stages and tracking
+the inputs and outputs of each run. SageMaker expects all inputs and outputs to
+be stored in S3, so the easiest way to integrate with DVC is to use [external
+dependencies and outputs].
+
+### Example: XGBoost pipeline
+
+For an example, see https://github.com/iterative/sagemaker-pipeline, which
+adapts an existing SageMaker tutorial from a notebook into a DVC pipeline. The
+first stage downloads the data and tracks the output so that it doesn't have to
+be re-downloaded on each run. We parametrize the `bucket` and `prefix` of the
+destination into a separate `params.yaml` file so they can be modified easily.
+The DVC pipeline stage is defined in `dvc.yaml` like this:
+
+```yaml
+prepare:
+  cmd:
+    - rm -f bank-additional.zip
+    - wget
+      https://sagemaker-sample-data-us-west-2.s3-us-west-2.amazonaws.com/autopilot/direct_marketing/bank-additional.zip
+    - python sm_prepare.py --bucket ${bucket} --prefix ${prefix}
+  deps:
+    - sm_prepare.py
+    - https://sagemaker-sample-data-us-west-2.s3-us-west-2.amazonaws.com/autopilot/direct_marketing/bank-additional.zip
+  outs:
+    - s3://${bucket}/${prefix}/input_data:
+        cache: false
+```
+
+The [preprocessing script] takes `bucket` and `prefix` as arguments and
+otherwise is copied directly from the original notebook code, which uses a
+SageMaker Processing job. The DVC pipeline stage tracks the command, scripts,
+input paths, and outputs paths, so that this stage will only be run again if any
+of those change:
+
+```yaml
+preprocessing:
+  cmd: python sm_preprocessing.py --bucket ${bucket} --prefix ${prefix}
+  deps:
+    - sm_preprocessing.py
+    - preprocessing.py
+    - s3://${bucket}/${prefix}/input_data
+  outs:
+    - s3://${bucket}/${prefix}/train:
+        cache: false
+    - s3://${bucket}/${prefix}/validation:
+        cache: false
+    - s3://${bucket}/${prefix}/test:
+        cache: false
+```
+
+Finally, the [training script] uses the SageMaker Estimator for XGBoost to train
+a model. We add all the model hyperparameters as arguments to make it easy to
+tune hyperparameters and track what changed. Hyperparameters are added under the
+`train` key in `params.yaml`. The DVC pipeline stage `cmd` includes `${train}`
+to pass all those arguments and track them as parameters, in addition to
```

Comment:
```suggestion
to [unpack and pass](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#dictionary-unpacking) all those arguments and track them as parameters, in addition to
```

Non blocking - but I think it's a vey non trivial behavior (and I wonder if we know how to create the right cli arguments in all cases 😄 )

Reply:
For better or worse, CLI args is how the sagemaker sdk and lots of other ml workflows expect to pass parameters, so we do our best to support it even though it is more prone to type errors and such than reading directly from a yaml file, for example.